src/main/java/com/kruize/environment/SupportedTypes.java

```diff
@@ -34,4 +34,7 @@
 
     public static final Set<String> MONITORING_AGENTS_SUPPORTED =
             new HashSet<>(Collections.singletonList("PROMETHEUS"));
+
+    public static final Set<String> POLICIES_SUPPORTED =
+            new HashSet<>(Arrays.asList("THROUGHPUT", "SECURITY","STARTUP"));
```

Comment:
This is causing the bug, because our policies can be a combination of these, like security-throughput, or security-startup, rather than just one of them. Please make this an ArrayList and change the logic in setPolicy() to set the policy to something like this:

```
for (String supportedPolicy : SupportedTypes.POLICIES_SUPPORTED)
        {
            if (policy.contains(supportedPolicy))
                this.policy = policy;
        }
```